exercises/practice/reverse-string/.docs/instructions.append.md

```diff
@@ -1,4 +1,4 @@
 # Instructions append
 
 For more help on how to solve this exercise, please refer to the tutorial provided as part of the hello world exercise:
-[TUTORIAL.md](https://github.com/exercism/java/blob/master/exercises/hello-world/TUTORIAL.md)
+[TUTORIAL.md](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/TUTORIAL.md)
```

Comment:
Same as the other one: we do not want to link to the github pages. Instead, we want to delete TUTORIAL.md and include the contents here directly.